UnitTests/GitUI.Tests/CommandsDialogs/RevisionDiffControllerTests.cs

```diff
@@ -186,15 +186,15 @@ public void BrowseDiff_OpenRevisionFile_Index()
         public void BrowseDiff_DeleteFile_WorkTree()
         {
             var rev = new GitRevision(ObjectId.WorkTreeId);
-            var selectionInfo = new ContextMenuSelectionInfo(rev);
+            var selectionInfo = new ContextMenuSelectionInfo(rev, allFilesOrDirectoryExist: true);
```

Comment:
How can this test have passed before? Only because of arbitrarily chosen default values for ctor arguments.

Add negative tests, please.

Reply:
These tests are meaningless... Updating though